docs/modules/ROOT/pages/how-tos/ingress/xff-configuration.adoc

```diff
@@ -0,0 +1,26 @@
+= Configuring X-Forwarded-For handling
+
+This how-to explains how to configure the ingress' handling of the forwarded headers (`Forwarded` and `X-Forwarded-For`) on a per-route basis. 
+
+== Possible behaviours
+
+* *`append`*: By default, the ingress is configured to append the headers to any existing headers. If `Forwarded` and `X-Forwarded-For` headers are already present when the ingress receives a request, the existing values are preserved, and new values are appended by the ingress.
+* *`replace`*: If forwarded header handling is set to `replace`, any existing forwarded headers are discarded. Only the headers set by the ingress will be available.
+* *`never`*: If forwarded header handling is set to `never`, the ingress will never set these headers, and leave any existing headers untouched.
+* *`if-none`*: If forwarded header handling is set to `if-none`, the ingress will only set the headers if they're not already present. If any headers are present, they're left unchanged.
+
+== Annotating the route
+
+Forwarded header handling is configured by annotating the `Route` object.
```

Comment:
Probably makes sense to note that the annotations can also be set on `Ingress` objects which are managed by the ingresstoroute controller